src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,14 @@
+import NextAuth from 'next-auth';
+import GithubProvider from 'next-auth/providers/github';
+
+export default NextAuth({
+  // Configure one or more authentication providers
+  providers: [
+    GithubProvider({
+      clientId: process.env.GITHUB_ID,
+      clientSecret: process.env.GITHUB_SECRET,
+    }),
```

Comment:
Could you replace this with a credential provider that doesn't require adding env var?